snapcraft/tests/test_common.py

```diff
@@ -23,6 +23,14 @@
 
 class CommonTestCase(tests.TestCase):
 
+    def setUp(self):
+        super().setUp()
+        # reset all default paths
+        common._plugindir = common._DEFAULT_PLUGINDIR
+        common._schemadir = common._DEFAULT_SCHEMADIR
+        common._librariesdir = common._DEFAULT_LIBRARIESDIR
+        common._tourdir = common._DEFAULT_TOURDIR
```

Comment:
Shouldn't this be reset in the tearDown instead?
You could do that nicely with something like:

self.addCleanup(self._reset_dirs), and move the four statements to the _reset_dirs method.